packages/trpc/server/routers/viewer/workflows.tsx

```diff
@@ -639,16 +648,25 @@ export const workflowsRouter = router({
                 }),
                 organizer: booking.user
                   ? {
-                      language: { locale: booking.user.locale || "" },
-                      name: booking.user.name || "",
-                      email: booking.user.email,
-                      timeZone: booking.user.timeZone,
+                      language: { locale: booking.user[0]?.locale || "" },
```

Comment:
I'm not a fan of polluting with ´booking.user[0]´ everywhere. It is not legible. Let's either set a constant (`const organizer = booking.user[0]`) or rethink where to save the organizer.